### DISCUSSION TITLE
Business Management and IT

### DISCUSSION THREAD

**Discussion by jessecomeau87** [community/community#83679] [Discussions]
(Business)
I am an Entrepreneur: Owner/Founder, CEO, Director, Partner of Ilios Selene (Private Firm), ISMC Corp. (Not for Profit), IIS Operations Division (Funding), Elysian Communications (Manufacturing). A Collective. 

(IT) 
Also a Cpu Systems Designer. On Codecademy learning C/C++, and Python with an IDE and Visual Studios. Getting SystemC, VCML, and Gem5 set up with GitHub. Most of the plugins and extensions are there. I want to get into Simulation and design my own Intelligent System. 

Our main industry is Business Management and IT. 

Is there anything missing? 

Looking forward from hearing from you. 

Jesse

© ISMC Corp. 2023

![ElysianCommDark.png](https://github.com/community/community/assets/110445461/4e36045a-73cb-4d89-93ae-de0067891e08)

![IISBlack.png](https://github.com/community/community/assets/110445461/5740f25e-13b1-422d-92c9-4db261e7f4a9)

![IliosSeleneWhite.png](https://github.com/community/community/assets/110445461/df0a605f-0db4-48a6-8360-7eb3887fa432)

![ISMCLogo_1.png](https://github.com/community/community/assets/110445461/bff29cd1-924c-4941-a4ea-1fb9b112cbda)

**Comment by Akash1134** [comment]
Hi @jessecomeau87 , we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
